### ISSUE TITLE
Keyboard access

### ISSUE THREAD

**Issue by mprazuch** [Inscryb/inscryb-markdown-editor#24]
Hello!
It's really good to see accessibility improvements toward keyboard access (buttons instead of links) but I've noticed there is still one thing missing to make it fully happen -  tabindex in below snippet is deliberitely set to "-1":

https://github.com/Inscryb/inscryb-markdown-editor/blob/3ac27873f189aabd2057dfae13057cfdc354ed61/src/js/inscrybmde.js#L119

Looks like quick fix but maybe there is any specific reason for it?

Kind regards,
mp.

**Comment by Gommorach** [comment]
The same issues exists in SimpleMDE, which this project is based on but that project isn't updated anymore.

I've made a pull requests that gives the option to change this behaviour 
#33

**Comment by CWharton** [comment]
Thanks for the PR @Gommorach.
